content/sensu-go/5.4/reference/agent.md

```diff
@@ -80,10 +80,15 @@ Any requests for unknown endpoints result in a 404 Not Found response.
 ### `/events` (POST)
 
 The `/events` API provides HTTP POST access to publish [monitoring events][7] to the Sensu backend pipeline via the agent API.
+The agent places events created via the `/events` POST endpoint into a queue, so that, in the event of a loss of connection with the backend, event data is preserved and the agent sends queued events to the backend once a connection is reestablished.
```

Comment:
```suggestion
The agent places events created via the `/events` POST endpoint into a queue, so that, in the event of a loss of connection with the backend, event data is preserved and the agent sends queued events to the backend once a connection is reestablished.

Queued events are stored on disk, so they are not lost in the event of an agent shutdown.
```

Reply:
Added a sentence to clarify that queued events are stored on disk.